scripts/install-elastio.sh

```diff
@@ -0,0 +1,229 @@
+#!/bin/bash
+
+me=$(basename $0)
+dir=$(readlink -f $(dirname $0))
+
+kernel_devel_install()
```

Comment:
This function name sounds generic, but it uses `yum` so it's CentOS/RHEL specific.  Don't you think that will be confusing to a future reader?

Reply:
Fedora too

Reply:
renamed kernel_devel_install -> cent_fedora_kernel_devel_install